mne/viz/_3d.py

```diff
@@ -1344,3 +1345,135 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_3d(dipole, trans, subject, subjects_dir=None, scale_factor=1e9,
+                   block=False, show=True):
+    """Plot dipoles in 3-D. Browse through the dipoles using mouse scroll or
+    up/down arrows.
```

Comment:
single line. More descriptive first line involving "mri", "slices", and/or "orthoview"; put interactivity instructions in separate paragraph